packages/web-config-server/src/apiV1/dataBuilders/generic/dataDownload/rawDataValues.js

```diff
@@ -83,10 +83,11 @@ class RawDataValuesBuilder extends DataBuilder {
             code => !excludeCodes.includes(code),
           );
         }
-        const { entityAggregation } = surveyConfig;
+        const { entityAggregation, entityIdToNameElements = {} } = surveyConfig;
```

Comment:
```suggestion
        const { entityAggregation, entityIdToNameElements } = surveyConfig;
```

We want to keep this undefined here - if not, [this check](https://github.com/beyondessential/tupaia/blob/1796-somoa-covid-rawdata-dowload/packages/web-config-server/src/apiV1/dataBuilders/DataBuilder.js#L85) will be true even if the field is not provided